src/components/Editor.jsx

```diff
@@ -63,8 +63,7 @@ class Editor extends React.Component {
   }
 
   _focusRequestedLine(requestedFocusedLine) {
-    if (get(requestedFocusedLine, 'component') !==
-      `editor.${this.props.language}`) {
+    if (get(requestedFocusedLine, 'componentKey') !== this.props.language) {
```

Comment:
it doesn't feel 100% right here to use language as the `componentKey`, but I wanted to get rid of all `editor.${this.props.language}`, which also feels a bit arbitrary. Please let me know if you have better ideas!